### PR TITLE
Release 2.1.0-beta

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

**Comment by breeswish** [comment]
LGTM

File: Cargo.lock [hunk 1]
```diff
@@ -1155,33 +1185,33 @@ name = "thread_local"
...
+version = "2.1.0-beta"
 dependencies = [
  "backtrace 0.2.3 (registry+https://github.com/rust-lang/crates.io-index)",
- "bitflags 1.0.1 (registry+https://github.com/rust-lang/crates.io-index)",
- "byteorder 0.5.1 (registry+https://github.com/rust-lang/crates.io-index)",
- "chrono 0.4.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "clap 2.23.3 (registry+https://github.com/rust-lang/crates.io-index)",
- "crc 1.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "crossbeam-channel 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
+ "bitflags 1.0.3 (registry+https://github.com/rust-lang/crates.io-index)",
```
Comment (overvenus) [inline Cargo.lock]: Why does it change so many crates?

**Pull request merged**